### ISSUE TITLE
Gallery page

### ISSUE THREAD

**Issue by meghprkh** [felicity-iiith/felicity18-website#3]

**Comment by saganshul** [comment]
I will recommend to put new photos this time. Last year photos are amazing. You can easily find them on DC++. Use them :)

**Comment by sounak98** [comment]
Should I work on the Gallery Page?

**Comment by meghprkh** [comment]
@sounak98 yeah. Look at the previous gallery page https://felicity.iiit.ac.in/gallery/ . We need not have something as fancy  for the placeholder site, but do use a library so that your work is easy.

**Comment by anubhabsen** [comment]
@sounak98  Updates?
